docs/content/operate/run-full-node.md

```diff
@@ -4,13 +4,32 @@
 
 The simplest way to run a full node is with the following commands:
 
-```bash
-# install the omni cli (or download https://github.com/omni-network/omni/releases/latest)
-curl -sSfL https://raw.githubusercontent.com/omni-network/omni/main/scripts/install_omni_cli.sh | bash -s
+1. Download and install the `omni` cli tool:
+   ```bash
+   # install the omni cli (or download https://github.com/omni-network/omni/releases/latest)
+   curl -sSfL https://raw.githubusercontent.com/omni-network/omni/main/scripts/install_omni_cli.sh | bash -s
+   ```
+2. Initialize `halo` and `geth` via Node Snapshot Mode or Full Sync Mode:
```

Comment:
lessismore, in the main "get a node running path", rather just pick a single flow. No need to complicate this quickstart with lots of different options and flows and stuff.

So include --node-snapshot in default flow.
Add a FAQ something like: --node-snapshot required more disk temporarily.

Main goal of docs is to provide manual process to load from snapshot in "Configure a Full Node"